mongodb-atlas-mixin/README.md

```diff
@@ -0,0 +1,103 @@
+
+## MongoDB Atlas operations overview
+The MongoDB Atlas operations overview dashboard provides a detailed look at the operations occurring in an Atlas node. Metrics include CRUD operation counts, connection counts, read and write operations and latency, deadlock counts, and lock wait times. Please note that lock related panels such as deadlocks and wait times will show "no data" until load is generated for their respective metrics.
+
+![First screenshot of the MongoDB Atlas operations overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/mongodb-atlas/screenshots/mongodb-atlas-operations-overview-1.png)
+![Second screenshot of the MongoDB Atlas operations overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/mongodb-atlas/screenshots/mongodb-atlas-operations-overview-2.png)
+
+## MongoDB Atlas elections overview
+The MongoDB Atlas elections overview dashboard provides a detailed look into the election states of replica sets in the Atlas cluster, at the node level. Metrics include various "called" and "successful" counts for various election types and catch-up related numbers like catch-ups successful, skipped, and failed.
+
+![First screenshot of the MongoDB Atlas elections overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/mongodb-atlas/screenshots/mongodb-atlas-elections-overview-1.png)
+![Second screenshot of the MongoDB Atlas elections overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/mongodb-atlas/screenshots/mongodb-atlas-elections-overview-2.png)
+
+## MongoDB Atlas sharding overview
+The MongoDB Atlas sharding overview dashboard is exclusive for sharded Atlas clusters. This dashboard includes metrics like refreshes started and failed, refresh times, blocked cache operations, stale configs, and counts for operations that target a different number of shards. Panels in this dashboard only apply to `shardsvr` and `mongos` nodes. Between those 2 node types, most panels only apply to one or the other.
+
+![First screenshot of the MongoDB Atlas sharding overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/mongodb-atlas/screenshots/mongodb-atlas-sharding-overview-1.png)
+![Second screenshot of the MongoDB Atlas sharding overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/mongodb-atlas/screenshots/mongodb-atlas-sharding-overview-2.png)
+
+## Environment configuration
+This mixin includes the MongoDB Atlas sharding overview dashboard, however the metrics used report data only when there is a sharded cluster in the project the prometheus integration is reporting on. Because of this, the MongoDB Atlas sharding overview dashboard is disabled by default. However, if you do have a sharded cluster and want to use the MongoDB Atlas sharding overview dashboard, you can set the `enableShardingDashboard` parameter in `config.libsonnet` to `true`.
```

Comment:
```suggestion
This mixin includes the MongoDB Atlas sharding overview dashboard, however the metrics used report data only when there is a sharded cluster in the project the prometheus integration is reporting on. Because of this, the MongoDB Atlas sharding overview dashboard is disabled by default. However, if you do have a sharded cluster and want to use the MongoDB Atlas sharding overview dashboard, you can set the `enableShardingOverview` parameter in `config.libsonnet` to `true`.
```
to match config.libsonnet

Reply:
good catch